esrally/mechanic/telemetry.py

```diff
@@ -776,17 +777,21 @@ def _store_merge_times(self, merge_times):
 
 
 class DiskIo(InternalTelemetryDevice):
```

Comment:
Unfortunately, we don't have a test yet for this class but I think it would be good to have one now. For the test I suggest you set `log_root` to the temporary directory (see `tempfile.mkdtemp()`).